Add a good remainder of Jamie Paige releases

uint32bit opened the pull request:
scope is not defined yet, to see a list of potential additions see [this google sheet](https://docs.google.com/spreadsheets/d/1QVeDg5XgvvJiFQMMOW-ekJUcRhv2QkK2zskbnADgADo/edit?usp=sharing)

god have mercy on me

To add:
- [x] Add Matryoshka -Club Mix-
- [x] Add Little Anger (so you take all these gifs and remove the data, and when you combine them you load the wav, reverse it and you get Little Anger, oh Jamie...)
- [x] Add smots gaming (?) https://www.youtube.com/watch?v=IOUoub2eGhc (does this count as a valid release, i don't think it's ever getting released anywhere else) (edit: lol lmao xd)
- [x] Add ON THE WIND
- [x] Add Video Hunting Specimen (Original Soundtrack)
- [ ] Add references 🪨🫷
- [ ] Add art tags
- [ ] Add Jamie's Page releases

To revise:
- [x] Revamp vocal synth credits
- [x] Revamp feature credits on early releases
- [x] Revise yt music video links
- [x] Consistent bandcamp descriptions (source credits(?))
- [ ] Credit Sources for MVs (?
- [ ] Unlink early aliases

uint32bit committed:
constant companions basic info

uint32bit committed:
constant companions lyrics check part 1

uint32bit committed:
constant companions lyrics check part 2

uint32bit commented on album/constant-companions.yaml:
```diff
@@ -830,6 +844,8 @@ Lyrics: |-
 
     (I swear on my life, I'll make it worth the while)
     (Never anything less than free!)
+    (???)
```
idk what it says here

uint32bit commented on album/constant-companions.yaml:
```diff
@@ -852,6 +868,13 @@ Lyrics: |-
+    If the sunlight stays behind us, out of sight
+    Be my object of affection, by candlelight
+
+    [??? type beat]
```
idk what it says here either

uint32bit commented on album/constant-companions.yaml:
```diff
@@ -931,23 +956,61 @@ Lyrics: |-
     All love and adoration
+    (Love will have a way)
     Oh, to be your constant companion!
+    (Love will have a way)
```
i did the backing vocals in these last 2 paragraphs in a way where it includes everything (sometimes 2 or 3 sentences are said all at once), lmk if it's all good

towerofnix replied:
This looks very good to me. :+1:

uint32bit committed:
Song of the Eared Robot

uint32bit committed:
Niklink i hope you don't mind me copying your format from the track above :)

uint32bit committed:
european moment

uint32bit committed:
asemic speech single
re-release for now, depending on consensus on lyrics the status can change

uint32bit committed:
constant companions singles

uint32bit committed:
jamies paige reference corrections

uint32bit committed:
music video urls "fixed"

uint32bit committed:
asemic speech single lyrics
yay

uint32bit committed:
🔥

uint32bit committed:
hope cafe
☕️MIKU MIKU BEEEEEEEEEEAAAAAAN☕️

uint32bit committed:
fix spacing on hope cafe commentary

uint32bit committed:
added missing lengths

uint32bit committed:
remove trailing spaces

uint32bit committed:
ride on time jamie paige arrange
a.k.a. my personal hell

uint32bit committed:
ride on time english version additional credits

uint32bit committed:
lowercase contributor credits

uint32bit committed:
using proper features
wiggle your fingers and jam the keys

uint32bit committed:
sax
sax

uint32bit committed:
radiant revival
Baby, do you know what you wanna hear?

uint32bit committed:
constant companions deluxe edition

uint32bit committed:
birdbrain single
baby- fuck

uint32bit committed:
ricedeity links and removed unnecessary link for puca o'donnel

uint32bit committed:
marlow jacobs links

uint32bit committed:
unit-0 links

uint32bit committed:
visualeyes links

uint32bit commented:
moved checkboxes to the main comment lol lmao xd

uint32bit committed:
Matryoshka -Club Mix- (bg and lyrics to be revised)

uint32bit commented:
oh yeah i also need to add the very obvious reference as this is a (re)mix
this is a reminder for myself

uint32bit committed:
matryoska references and revised lyrics
sorry i coudln't find a good background, like there's cool art assets but they're only in the video and a little bit in the cover

uint32bit committed:
you'd think i'd check running the wiki before commiting (plus misc fixes)

uint32bit committed:
you're teling me a shrimp fried this music video?

uint32bit committed:
constant companions mvs!
so can we wander for a spell

uint32bit committed:
fire and cc deuxe mvs
a benefit of having put all the artsts of machine love is that all of these artists are already represented here

uint32bit committed:
radiant revival mv

uint32bit committed:
matryoshka mv
oh yeah i almost forgot one of the reasons this feature exists in the first place

uint32bit committed:
mistakes were made

uint32bit committed:
additional links and credits

uint32bit committed:
wait, those were music videos, THAT DOESN'T COUNT!!!

uint32bit committed:
feat.

uint32bit committed:
little anger
unfortunately idk who made da bird.gif so it's not making it in yet

uint32bit committed:
smots gaming
and let me hear some morre of that song

uint32bit committed:
important tags and commentary for smots gaming

uint32bit committed:
The Great Vocal Synth Replacement™

uint32bit committed:
little anger tweaks

uint32bit committed:
smots gaming cover fix

uint32bit committed:
smots gaming references smots gaming

uint32bit committed:
corrected little anger lyrics based on c-sides

uint32bit committed:
smots credits reflecting c-sides release

uint32bit committed:
fix smots

uint32bit committed:
one small moment of silence + solaria rename

uint32bit committed:
fix features on re-releases

uint32bit committed:
radiant revival lyrics fix

uint32bit committed:
rename Nancy! to Catherine de to recent credits
(i still think this name is a joke and she still uses Nancy herself but last art credits are consistently this so... ¯\_(ツ)_/¯

uint32bit committed:
c-sides
'cause all you hear's the start of a ringtone

uint32bit committed:
somebody once told me the lyrics to object of affection

thank u trinity

uint32bit committed:
whoops

uint32bit committed:
let's do the background baby

uint32bit committed:
love renewed bg

uint32bit committed:
pee pee pee pee bg

uint32bit committed:
cc+ccde commentary and crediting sources for the main album pages

uint32bit committed:
that's the wrong album linked
bit what are you doing!!!???

uint32bit committed:
c-sides commentary revision

uint32bit committed:
smots & osmos lyrics

uint32bit committed:
CCDE og lyrics & crediting sources

uint32bit committed:
constant companions + little anger crediting sources and original lyrics
WHAT DO YOU MEAN RE-RELEASES DON'T INHERIT THESE????

uint32bit committed:
CCDE crediting sources & lyric sources because they're not inherited

uint32bit committed:
cadmium colors single crediting changes, lyrics and commentary tweaks

uint32bit committed:
cadmium colors single fix & machine love single credits/commentary/lyrics

uint32bit committed:
fire og lyrics and crediting sources

uint32bit committed:
hope cafe og lyrics and crediting sources

uint32bit committed:
matryoshka club mix og lyrics and crediting sources

uint32bit committed:
radiant revival crediting sources + matroshka fix

uint32bit committed:
fix urls and mind binding being doubled for some reason

uint32bit committed:
ride on time crediting sources

uint32bit committed:
le epic link you can click

uint32bit committed:
women wrestling crediting sources

uint32bit committed:
love renewed crediting sources

uint32bit committed:
encore crediting sources

uint32bit committed:
people posture play pretend crediting sources

uint32bit committed:
ebichahan crediting sources

uint32bit committed:
ebichahan fixed sources and added missing credit

uint32bit committed:
bittersweet credit sources and fixed commentary

uint32bit committed:
ebichahan cool and new music video

uint32bit committed:
anew again crediting sources and updated commentary

uint32bit committed:
excerpt

uint32bit committed:
autumn every day commentary and crediting sources

uint32bit committed:
virtue and destiny commentary update + crediting sources

uint32bit committed:
clickable links on c-sides

uint32bit committed:
missing destiny credits

towerofnix committed:
sort artists

sorry for not getting this right during the rebase prolly lol

uint32bit committed:
on the wind

towerofnix committed:
Exclude From Other Group Album Summaries

uint32bit committed:
video hunting specimen

uint32bit committed:
Merge branch 'preview' into bit/jamie-paige

uint32bit committed:
workit fixes

uint32bit committed:
quick jerma samples plus some other obvious ones
